### PR TITLE
Provide examples for the wikify widget

### PR DESCRIPTION
**Pull request by DesignThinkerer**
This PR add examples for using the wikify widget. Discussion: https://talk.tiddlywiki.org/t/doc-wikifywidget-examples-usecase/5890 

---
<small>Submitted using https://saqimtiaz.github.io/tw5-docs-pr-maker/.</small>

### CODE REVIEW

**Comment by pmario** [comment]
I think the first example should be something **that works**. It is OK to show how we should **not** do it, but it needs much more explanation, what we want to show. 

If the first example is a **fail** users think it is broken. If we want to show "failed examples"  it needs a detailed description, what's going on.

**Comment by pmario** [comment]
I also think, the example Output code should be more readable. Something similar to this.

image

**Comment by DesignThinkerer** [comment]
> I also think, the example Output code should be more readable. Something similar to this.
> 
> image

Thanks for the review. I applied your suggestions :)

**Comment by pmario** [comment]
I think it looks more readable now. But the https://deploy-preview-8792--tiddlywiki-previews.netlify.app/#TestCases%2FWikifyWidget%2FWidgetValue%2FSuccess fails. 

You need to remove the very last new-line in the file - then it works

```
title: ExpectedResult

<p>
    <span style="color:white;background:purple;">
        white text on purple is a good contrast
    </span>
</p>    <- The line break here needs to be removed
```

File: editions/tw5.com/tiddlers/TestCases_WikifyWidget_RenderHTML.tid [hunk 1]
```diff
@@ -0,0 +1,28 @@
+created: 20241201235747500
+modified: 20241202014214626
```
Comment (btheado) [inline editions/tw5.com/tiddlers/TestCases_WikifyWidget_RenderHTML.tid:2]: This test case is missing a description.

**Comment by Jermolene** [comment]
Great stuff, thank you @DesignThinkerer

**Pull request merged**